registry/registry_internals.c

```diff
@@ -323,3 +323,49 @@ char *registry_get_this_machine_guid(void) {
 
     return guid;
 }
+
+char *registry_get_mgmt_api_key(void) {
```

Comment:
I think it is not right to put this in `registry`.
When I said "check what the registry does" I didn't mean "put the management API key code at the registry".

:)

Reply:
Moved to web_api_v1.c and fixed, renamed